packages/layer1-usdc/schema.graphql

```diff
@@ -0,0 +1,21 @@
+type MessageReceived @entity(immutable: true) {
+  id: Bytes!
+  caller: Bytes! # address
+  sourceDomain: BigInt! # uint32
+  nonce: BigInt! # uint64
+  sender: Bytes! # bytes32
+  messageBody: Bytes! # bytes
+  blockNumber: BigInt!
+  blockTimestamp: BigInt!
+  transactionHash: Bytes!
+}
+
+type MessageSent @entity(immutable: true) {
+  id: Bytes!
+  message: Bytes! # bytes
+  blockNumber: BigInt!
+  blockTimestamp: BigInt!
+  transactionHash: Bytes!
+  sender: String! # address
```

Comment:
It's more efficient to have sender be Bytes type

Reply:
Good point, also make the code easier to read